addons/survey/data/survey_demo_media.xml

```diff
@@ -0,0 +1,234 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This could be cool to merge MyCompany Vendor Certification and quiz about tree.
Modify MyCompany Vendor Certification with stuff about trees (you can keep your beautiful data here)
so we keep the 'demo data answers'

Reply:
Finally, I merged it with Quiz about our Company. To be honest, I was lacking inspiration to adapt MyCompany Vendor Certification with stuff about trees and keeping the sensation of a certification in the same time. I think it makes more senses with Quiz about our Company which already asks some questions about wood and trees.